src/System.Runtime.WindowsRuntime/src/System/InternalHelpers.cs

```diff
@@ -1,16 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Runtime.InteropServices;
-
-namespace System
-{
-    internal static class InternalHelpers
-    {
-        internal static void SetErrorCode(this Exception ex, int code)
-        {
-            InteropExtensions.SetExceptionErrorCode(ex, code);
```

Comment:
This delete isn't sufficient to allow us to remove `[assembly: InternalsVisibleTo("System.Runtime.WindowsRuntime, PublicKey=00000000000000000400000000000000", AllInternalsVisible = false)]` from corelib I assume..

Reply:
Right. It is getting us a bit closer though.